src/redis_manager.py

```
@@ -65,9 +65,11 @@ class RedisQueue:
     def __init__(self):
```

Ruff (0.15.15)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)